src/status/status.py

```diff
@@ -21,6 +21,10 @@
 AUTH_HEADER_NAME = 'Authorization'
 
 SYSTEMS_PUBLIC  = os.environ.get("SYSTEMS_PUBLIC").strip('\'"').split(";")
+# ; separated for system (related with SYSTEMS_PUBLIC length, and for each filesystem mounted inside each system, separated with ":")
+# example: let's suppose SYSTEMS_PUBLIC="cluster1;cluster2", cluster1 has "/fs-c1-1" and "/fs-c1-2", and cluster2 has mounted "/fs-c2-1":
+# FILESYSTEMS = "/fs-c1-1,/fs-c1-2;fs-c2-1"
+FILESYSTEMS = os.environ.get("FILESYSTEMS").split(";")
```

Comment:
Add "strip" function:

FILESYSTEMS = os.environ.get("FILESYSTEMS").strip('\'"').split(";")

Right now "parameters" is giving the following result:

'{"description":"Firecrest\'s parameters","out":{"storage":[{"name":"OBJECT_STORAGE","unit":"","value":"\'s3v4\'"},{"name":"STORAGE_TEMPURL_EXP_TIME","unit":"seconds","value":"604800"},{"name":"STORAGE_MAX_FILE_SIZE","unit":"bytes","value":"5368709120"},{"name":"FILESYSTEMS","unit":"","value":[{"mounted":[**"\\"/home"**],"system":"system01"},{"mounted":[**"/home\\""**],"system":"system02"}]}],"utilities":[{"name":"UTILITIES_MAX_FILE_SIZE","unit":"MB","value":"5"},{"name":"UTILITIES_TIMEOUT","unit":"seconds","value":"5"}]}}\n'

Reply:
Updated in following commit